device-types/Arista/AWE-7250R-16S-F.yaml

```diff
@@ -0,0 +1,55 @@
+---
+manufacturer: Arista
+model: AWE-7250R-16S-F
+slug: arista-awe-7250r-16s-f
+part_number: AWE-7250R-16S-F
+u_height: 2
+is_full_depth: true
+airflow: front-to-rear
+weight: 13.6
+weight_unit: kg
+comments: '[Arista AWE-7250R-16S-F Datasheet](https://www.arista.com/assets/data/pdf/Datasheets/AWE-7200R-Datasheet.pdf)'
+console-ports:
+  - name: Console
+    type: rj-45
```

Comment:
What do you think about adding `label: IOIOI` to the Console ports (in general) and `label: <⋯>` to the Management ports (again, in general)?

Ends up looking like this for instance:

![image](https://github.com/netbox-community/devicetype-library/assets/410018/d24cdd5e-ea65-418a-8515-62f2308d2ed3)

Reply:
I don't have any strong feelings one way or the other. Do you think its helpful/useful?

Reply:
@Supermathie if you would like to see this implemented, please feel free to create a new PR to add this. Thank you